.travis.yml

```diff
@@ -4,7 +4,9 @@ services:
   - postgresql
 
 before_script:
-  - psql -U postgres -c 'CREATE DATABASE chitter_test;'
+  - psql -U postgres -c 'CREATE DATABASE chitter_challenge_test;'
+  - psql -U postgres -d chitter_challenge_test -f ./db/migrations/01_create_account_table.sql
+  - psql -U postgres -d chitter_challenge_test -f ./db/migrations/02_create_peeps_table.sql
```

Comment:
oh... I wonder if this will make your travis not failing?? still pending tough, but I'll keep on eye :D